File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/05.While-Loop/01-lab/05.accountBalance.js
function accountBalance(input) {

    let index = 0;
    let element = input[index];

    let total = 0;

    while (element !== "NoMoreMoney") {
        element = Number(input[index]);
        if (element < 0) {
            console.log("Invalid operation!");
            break;
        }
        console.log("Increase: " + element.toFixed(2));
        total += element;
        index++;
        element = input[index];
    }

    console.log("Total: " + total.toFixed(2));
}
accountBalance(["5.51", "69.42", "100", "NoMoreMoney"]);

File: Python_Basics/Homeworks-And-Labs/05.while_loop/01.lab/05.account_balance.py
element = input()

total = 0

while element != "NoMoreMoney":
    element = float(element)
    if element < 0:
        print("Invalid operation!")
        break

    print(f'Increase: {element:.2f}')
    total += element
    element = input()

print(f'Total: {total:.2f}')